cypress/support/commands.js

```diff
@@ -47,24 +48,39 @@ Cypress.Commands.add('login', (email, username, password) => {
   });
 });
 
-Cypress.Commands.add('createArticle', (title, description, body) => {
-  cy.getCookie('auth').then((token) => {
-    const authToken = token.value;
+Cypress.Commands.add('createArticle', (user, article) => {
+  cy.request({
+    method: 'POST',
+    url: '/api/users/login',
```

Comment:
The URL '/api/users/login' is used for logging in the user. Ensure that this endpoint is correct and matches the API specification for user login.